.github/workflows/main-drivers.yml

```diff
@@ -0,0 +1,59 @@
+name: Collector Drivers CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  init:
+    uses: ./.github/workflows/init.yml
+    secrets: inherit
+    with:
+      cpaas-workflow: false
+
+  build-drivers:
+    uses: ./.github/workflows/drivers.yml
+    with:
+      drivers-bucket: ${{ needs.init.outputs.drivers-bucket }}
+      bundles-bucket: ${{ needs.init.outputs.bundles-bucket }}
+      branch-name: ${{ needs.init.outputs.branch-name }}
+    needs: init
+    secrets: inherit
+
+  upload-drivers:
+    uses: ./.github/workflows/upload-drivers.yml
+    with:
+      gcp-bucket: ${{ needs.init.outputs.push-drivers-bucket }}
+      merged-drivers-bucket: ${{ needs.init.outputs.merged-drivers-bucket }}
+    if: ${{ needs.build-drivers.outputs.parallel-jobs > 0 }}
+    needs:
+    - init
+    - build-drivers
+    secrets: inherit
+
+  check-drivers-build:
+    uses: ./.github/workflows/check-drivers-failures.yml
+    needs: build-drivers
+    if: ${{ needs.build-drivers.outputs.parallel-jobs > 0 }}
+    secrets: inherit
+    with:
+      logs-artifact: driver-build-failures-*
+
+  build-support-packages:
+    uses: ./.github/workflows/support-packages.yml
+    with:
+      branch-name: ${{ needs.init.outputs.branch-name }}
+      upstream-drivers-bucket: ${{ needs.init.outputs.drivers-bucket }}
+      downstream-drivers-bucket: ${{ needs.init.outputs.cpaas-drivers-bucket }}/x86_64
+      support-packages-bucket: ${{ needs.init.outputs.support-packages-bucket }}
+      support-packages-index-bucket: ${{ needs.init.outputs.support-packages-index-bucket }}
+      public-support-packages-bucket: ${{ needs.init.outputs.public-support-packages-bucket }}
+    if: |
+      always() &&
```

Comment:
Is `always()` needed here?

Reply:
It's needed, there are some cases were steps are skipped but the support packages still need to be built and this ensures that (can't recall the exact case where not having it cause issues, but I did get it not so long ago).